install_config/storage_examples/ceph_example.adoc

```diff
@@ -12,34 +12,41 @@ toc::[]
 
 == Overview
 
-For an explanation of persistent volumes (PVs), persistent volume claims (PVCs), and
-using Ceph RBD as persistent storage, please see the section on
-link:../persistent_storage/persistent_storage_ceph_rbd.html[Persistent Storage Using Ceph Rados Block Device].
-Below, is an end-to-end example of using an existing ceph cluster as an OpenShift persistent store.
-It is assumed that a working ceph cluster has already been setup. If not please read more about 
-link:https://access.redhat.com/products/red-hat-ceph-storage[ceph here].
+This topic provides an end-to-end example of using an existing Ceph cluster as
+an {product-title} persistent store. It is assumed that a working Ceph cluster
+is already set up. If not, consult the
+link:https://access.redhat.com/products/red-hat-ceph-storage[Overview of Red Hat
+Ceph Storage].
+
+link:../persistent_storage/persistent_storage_ceph_rbd.html[Persistent Storage
+Using Ceph Rados Block Device] provides an explanation of persistent volumes
+(PVs), persistent volume claims (PVCs), and using Ceph RBD as persistent
+storage.
 
 [NOTE]
 ====
-All `oc ...` commands are executed on the OpenShift master node.
+All `oc ...` commands are executed on the {product-title} master node.
 ====
```

Comment:
Same nit from the other PR. Should master node be "master host" or "all-in-one host"?